test/stale.test.js

```diff
@@ -195,4 +195,30 @@ describe('stale', () => {
       expect(stale.getClosable).not.toHaveBeenCalled()
     }
   )
+
+  test(
+    'should count weekends for age by default',
+    async () => {
+      let stale = new Stale(github, {perform: true, owner: 'probot', repo: 'stale', logger: robot.log})
+      stale.config.skipWeekends = false
+      const now = Date.now()
+      Date.now = jest.genMockFunction().mockReturnValue(now)
+      const sinceDate = stale.since(7)
+      expect(sinceDate).toEqual(new Date(now - 7 * 24 * 60 * 60 * 1000))
+    }
+  )
+
+  test(
+    'should not count weekends for age when configured with skipWeekends',
+    async () => {
+      let stale = new Stale(github, {perform: true, owner: 'probot', repo: 'stale', logger: robot.log})
+      stale.config.skipWeekends = true
+
+      const now = new Date(2018, 5, 18) // June 18, 2018, a Monday
+      Date.now = jest.genMockFunction().mockReturnValue(now)
+
+      const sinceDate = stale.since(1)
```

Comment:
I understand this test and it makes sense, but can you add another test that shows that this actually works overall as opposed to just the `since()` method

Reply:
I'm happy to do this, but I could use some guidance here. I don't see any tests that determine a set of closable issues, and I'm not really sure the right place to mock an API response. I could assert that `search()` uses (or `getStale()` and `getClosable()`) use the older date, but I'm not sure that would really provide the level of confidence you're looking for?